### PR TITLE
{bio}[GCC/11.3.0,GCCcore/11.3.0,foss/2022a] edlib v1.3.9, duplex-tools v0.3.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node092.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7fce69aa5e9ce446e3b6e041f5b594b1 for a full test report.

**Comment by casparvl** [comment]
Synced with develop, since an identical `minimap2` was just merged 2 hours ago...

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn260.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/6311957341e407b0e27ba07f04627c73 for a full test report.

File: easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+homepage = 'https://github.com/nanoporetech/duplex-tools'
+description = """
+Duplex Tools contains a set of utilities for dealing with Duplex sequencing data. Tools are provided
+ to identify and prepare duplex pairs for basecalling by Dorado (recommended) and Guppy, and for
+ recovering simplex basecalls from incorrectly concatenated pairs.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+_minimap2_ver = '2.24'
```
Comment (casparvl) [inline easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb:16]: Why do you define this? It's only used once, in the dependencies list, right? Might as well hardcode it there then.

**Review by casparvl — CHANGES_REQUESTED** [review]

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb:16]: damn, injecting checksums overwrote the use of it in `exts_list`

File: easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    ('tqdm', '4.64.0'),
+    ('Arrow', '8.0.0'),
+    ('h5py', '3.7.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('mappy', '2.24', {
```
Comment (jfgrimm) [inline easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb:34]:
```suggestion
    ('mappy', _minimap2_ver, {
```

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn499.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/807fd9017ce280abf88c8320ce714875 for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**